models/app.models.js

```diff
@@ -83,4 +83,20 @@ exports.updateArticle = (votes, id) => {
         }
         return result.rows[0]
     })
+}
+
+exports.removeComment = (id) => {
+    return db.query(`DELETE FROM comments
+    WHERE comment_id = $1;`, [id]).then((result) => {
```

Comment:
could have added a `RETURNING *` to this command to check whether it existed or not. Good that you found a solution though.